### PR TITLE
fix(tests): Replace `has...()` methods with `get...()` methods

### PR DESCRIPTION
**Pull request by jarifibrahim**
This PR fixes https://github.com/fabric8-ui/fabric8-planner/issues/2585

### CODE REVIEW

**Comment by nimishamukherjee** [comment]
@jarifibrahim is this PR complete?

**Comment by jarifibrahim** [comment]
@nimishamukherjee Yes. It's complete.

File: src/tests/ui/planner/workitem-quickpreview.ts [hunk 1]
```diff
@@ -200,72 +200,68 @@ export class WorkItemQuickPreview extends ui.BaseElement {
...
+  async getArea() {
     await this.loadingAnimation.untilCount(0);
     let area = await this.areaDropdown.getTextWhenReady();
-    this.debug("Expect area: " + area + " to be " + areaName);
-    return area === areaName;
+    return area;
   }
 
-  async hasCreator(name: string): Promise<Boolean> {
+  async getCreator() {
```
[the remark this answers is not in the document]
Comment (jarifibrahim) [reply src/tests/ui/planner/workitem-quickpreview.ts]: @Raunak1203 How would you get the `ElementArrayFinder`? All the element finders are defined in their respective page object file, the `specs` files do not know what selector to use.

[the remark this answers is not in the document]
Comment (Raunak1203) [reply src/tests/ui/planner/workitem-quickpreview.ts]: @jarifibrahim u can use something like this: See if this works

         getAssignees(){
           return this.gGetElements(this.assigneeDiv);
         }

**Comment by jarifibrahim** [comment]
[test]

File: src/tests/ui/planner/workitem-quickpreview.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (jarifibrahim) [reply src/tests/ui/planner/workitem-quickpreview.ts]: As discussed with @Raunak1203, we'll be keeping the functions as they are right now. Changing them to some other generic form won't help.

**Pull request merged**